src/test/rustdoc/type-layout.rs

```diff
@@ -2,6 +2,7 @@
 
 // @has type_layout/struct.Foo.html 'Size: '
 // @has - ' bytes'
+// @matches - '<h2 id="layout".+Layout.+a href="#layout" class="anchor">'
```

Comment:
This is quite tricky to read. Instead I suggest:

```suggestion
// @has - '//*[@id="layout"]/a[@href="#layout"]' ''
```

Reply:
If you want to check the layout text (which is a good idea!), you can also add:

```
// @has - '//*[@id="layout"]' 'Layout'
```